Code/RDBoost/Wrap.h

```diff
@@ -18,6 +18,10 @@
 //
 #include <boost/python.hpp>
 #include <boost/python/stl_iterator.hpp>
+#include <memory>
+// ha ha... we don't actually want to use this, but we need to keep support old
+// C++ versions:
+#define rdk_unique_ptr std::auto_ptr
```

Comment:
auto_ptr and unique_ptr have very similar semantics, and APIs. If the plan is to eventually go C++11 maybe having an ifdef here for c++11 and just switching to unique_ptr is an option? The con to using auto_ptr is that it's deprecated.